### PR TITLE
some updates to docs

### PR DESCRIPTION
**Pull request by williamjmorenor**
1- Reference support of python3.5
2- Reference third party themes available (material, alabaster, cinder)

Signed-off-by: William Moreno Reyes <williamjmorenor@gmail.com>

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ $ pip --version
 pip 1.5.2
 ```
 
-MkDocs supports Python 2.6, 2.7, 3.3 and 3.4.
+MkDocs supports Python 2.6, 2.7, 3.3, 3.5 and 3.5.
```
Comment (waylan) [inline docs/index.md]: You might want to change that to "3.4 and 3.5" rather than "3.5 and 3.5"

File: docs/user-guide/styling-your-docs.md [hunk 2]
```diff
@@ -100,6 +100,30 @@ the [Bootswatch] project.
 
 ![Yeti](http://bootswatch.com/yeti/thumbnail.png)
 
+## Third party themes
+
+There are many themes provides for the community like:
```
Comment (waylan) [inline docs/user-guide/styling-your-docs.md]: This is a nitpick, but I wouldn't use "many" here. Also, "provided" not "provides".

**Comment by waylan** [comment]
I thought the decision was to list these in the wiki rather in the docs. See #818.

**Comment by d0ugal** [comment]
Yup, happy to take the 3.5 addition in a new PR. Listing themes in the docs isn't sustainable - so we should go ahead with the plan discussed in #818.

**Pull request closed without merging**